lib/facebook.ex

```diff
@@ -213,6 +213,77 @@ defmodule Facebook do
     Facebook.Graph.get(~s(/#{page_id}/feed), params)
   end
 
+  @doc """
+  Gets the total number of people who liked an object.
+  An *object* stands for: post, comment, link, status update, photo.
+
+  If you want to get the likes of a page, please see *pageLikes*.
+
+  ## Example
+      iex> Facebook.objectCount(:likes, "1326382730725053_1326476257382367", "<Token>")
+      2
+
+  See: https://developers.facebook.com/docs/graph-api/reference/object/likes
+  """
+  def objectCount(:likes, object_id, access_token) do
+    :likes
+      |> Atom.to_string
+      |> _objectSummary(object_id, access_token)
+      |> _summaryCount
+  end
+
+  @doc """
+  Gets the total number of people who commented an object.
+  An *object* stands for: post, comment, link, status update, photo.
+
+  ## Example
+      iex> Facebook.objectCount(:comments, "1326382730725053_1326476257382367", "<Token>")
+      2
+
+  See: https://developers.facebook.com/docs/graph-api/reference/object/comments
+  """
+  def objectCount(:comments, object_id, access_token) do
+    :comments
+      |> Atom.to_string
+      |> _objectSummary(object_id, access_token)
+      |> _summaryCount
+  end
+
+  """
+  Provides the summary of a GET request when the 'summary' query parameter is
+  set to true.
+
+  ## Example
+      iex> _objectSummary("comments", "1326382730725053_1326476257382367", "<Token>")
+      %{"total_count" => 47}
+  """
+  defp _objectSummary(scope, object_id, access_token) do
```

Comment:
please remove the `_` in front of the function name. The function is anyway private due to `defp` :)